### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by Samuellekpor**
In this first milestone, I: 

1. Create the database vet_clinic.
2. Create the table 'animals'.
3. Insert animals data;
4. Write queries for specific quests.

Attached is the screenshot of the queries results:
animals-querries-results

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```
Comment (thecodechaser) [inline queries.sql:5]: - [ ] The second query is not running properly and not giving results, as I can see in the screenshot. Please make sure that it runs without any issues and produces the expected results.
  - You can also try with `SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;`.

image

- [ ] The screenshot is missing the result of the last query. `SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;` Please make sure to add screenshot of the result of this query in the PR.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Samuellekpor👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] Please use the correct name `Vet Clinic` for the project name in the README file, Currently, it's `curriculum-databases-projects-template`. 

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Samuellekpor) [reply queries.sql:5]: Hi, @thecodechaser !!!
That line was an error. The second query is actually after that one. It is the one in the queries.sql file: 
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'.

Comment (thecodechaser) [reply queries.sql:5]: @Samuellekpor Ohh, I didn't notice that. Please ignore this change and only work on the second change and README change I requested and go for another review.

**Review by uwadonat — APPROVED** [review]
## Hi @Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good commit messages ✔️
- Professional readme file ✔️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by thecodechaser** [comment]
@Samuellekpor Could you please check your slack for my messages

File: queries.sql (hunk 1 above)
Comment (Samuellekpor) [reply queries.sql:5]: Alright! Thank you!!

**Pull request merged**